axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineServiceImpl.java

```diff
@@ -464,4 +470,28 @@ protected List<MoveLine> getMoveLinesFromIds(List<Integer> moveLineIds) {
     }
     return moveLineList;
   }
+
+  @Override
+  public String computeMoveLineTaxLineSetDomain(int functionalOriginSelect, String dateString) {
+    String[] split = dateString.split("-");
+    LocalDate date =
+        LocalDate.of(
+            Integer.parseInt(split[0]), Integer.parseInt(split[1]), Integer.parseInt(split[2]));
+
+    String domain = "(self.endDate > :date OR self.endDate IS NULL) ";
+    if (functionalOriginSelect == 3) {
```

Comment:
Same as before, the repository variable MoveRepository.FUNCTIONAL_ORIGIN_SALE should be used instead of 3

Reply:
This method is not used in this ticket, so there is no way to test it. I will put this modification in RM#83398.